R/assocTestAggregate.R

```diff
@@ -225,8 +209,6 @@ setMethod("assocTestAggregate",
         }
     } else {
         # user supplied weights
-        weight <- weight[keep]
-        
```

Comment:
@smgogarten -- I think this `weight0` check could be build into the `.prepGenoBlock` function. Add a check on `if(!is.null(weight))` at line 60 of that function. Then we'd only have to do the subsets on `keep` once, instead of doing it again here. (double check my logic of the order of things)

Reply:
Good idea, I've done that